recipe/recipe_test.go

```diff
@@ -104,11 +104,10 @@ func TestGetRecipe(t *testing.T) {
 
 		Convey("Then whe GetRecipe is called, one GET /recipes/ID call is performed and the expected error is returned", func() {
 			recipe, err := recipeClient.GetRecipe(ctx, testUserAuthToken, testServiceToken, recipeID)
-			So(err, ShouldResemble, &Error{
-				err:        errors.New("failed to unmarshal response body: invalid character 'i' looking for beginning of value"),
-				statusCode: http.StatusInternalServerError,
-				logData:    log.Data{"response_body": "invalidRecipeBody"},
-			})
+			So(err, ShouldResemble, dperrors.New(
+				errors.New("failed to unmarshal response body: invalid character 'i' looking for beginning of value"),
+				http.StatusInternalServerError,
+				log.Data{"response_body": "invalidRecipeBody"}))
```

Comment:
bracket should be closed on the next line (keep in line with where scope was opened)